assets/tests/abac_tests.yaml

```diff
@@ -586,6 +911,66 @@ tests:
             context:
               "y": 10
             errorCode: 2000
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              relation: viewer
+              object: document:1
+            temporarilySkipReason: "partially applied condition param behavior still needs to be defined"
+          - request:
+              filters:
+                - user
+              relation: viewer
+              object: document:1
+            temporarilySkipReason: "partially applied condition param behavior still needs to be defined"
```

Comment:
The intended behavior of this test case along with those adjacent needs to be defined.